update-refactor.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+platform=$(cat /etc/refactor.version | awk -F '\n' '{print $3}')
+
+source BaseLinux/${platform}/Linux
+
+ANSIBLE_PLATFORM_VARS=$(sed 's/"update":"false"/"update":"true"/g' <<< ${ANSIBLE_PLATFORM_VARS})
+
+sudo systemctl stop klipper
+
+if [[ ${platform} == 'replicape' ]]; then
+    sudo systemctl stop klipper_pru
+fi
+
+sudo mount -o remount,rw /boot/
+sudo ansible-playbook -T 180 --extra-args ${ANSIBLE_PLATFORM_VARS}
+sudo mount -o remount,r /boot/
+
+if [[ ${platform} == 'replicape' ]]; then
+    sudo systemctl start klipper_pru
+fi
+sudo systemctl start klipper
```

Comment:
Missing newline

Reply:
Instead of a script that requires the user to download the repository before running the script, could we have the script download the repository first?

Reply:
Good point! Indeed, that seems like a more sensible idea. Having the script as  `/home/debian/update-refactor.sh`  that then clones the repo to /tmp/ so it gets erased after the update reboots?